modules/search/instant-search/lib/query-string.js

```diff
@@ -127,14 +127,14 @@ export function getFilterQuery( filterKey ) {
 }
 
 // These filter keys have been activated/selected outside of the overlay sidebar
-export function getPreselectedFilterKeys( overlayWidgets ) {
-	return getUnselectableFilterKeys( overlayWidgets ).filter(
+export function getPreselectedFilterKeys() {
+	return getUnselectableFilterKeys().filter(
 		key => Array.isArray( getFilterQueryByKey( key ) ) && getFilterQueryByKey( key ).length > 0
 	);
 }
 
-export function getPreselectedFilters( widgetsInOverlay, widgetsOutsideOverlay ) {
-	const keys = getPreselectedFilterKeys( widgetsInOverlay );
+export function getPreselectedFilters( _, widgetsOutsideOverlay ) {
```

Comment:
Is the `_` argument so we can keep the existing function signature? Might be worth a comment explaining that I think.

Reply:
Correct, this is the defacto solution suggested (and auto-fixed) by ESLint. I went ahead and added a comment.